core/templates/dev/head/pages/exploration-editor-page/services/user-email-preferences.service.spec.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the UserEmailPreferencesService.
+ */
+
+// TODO(#7222): Remove the following block of unnnecessary imports once
+// the code corresponding to the spec is upgraded to Angular 8.
+import { UpgradedServices } from 'services/UpgradedServices';
+// ^^^ This block is to be removed.
+
+// eslint-disable-next-line max-len
+require('pages/exploration-editor-page/services/user-email-preferences.service.ts');
```

Comment:
@Arnesh07, why did you use `eslint-disable-next-line max-len` here? you could write the require statement in two lines?

Reply:
@nishantwrp Yes I can, I have seen both kinds of usage in the codebase. I prefer this one, no particular cause.

Reply:
@Arnesh07 Please see #6446. I think we should avoid using eslint wherever we can.
Thanks

Reply:
Okay, I have made the change. @nishantwrp